src/main/java/duke/interaction/Ui.java

```diff
@@ -0,0 +1,33 @@
+package duke.interaction;
+
+import duke.common.DukeString;
+
+/**
+ * A class that handles the output to user.
+ */
+public class Ui {
+    private Ui() {
+    }
+
+    /**
+     * Prints the given message with a separator.
+     *
+     * @param msg the message to be printed.
+     */
+    public static void printOut(final String msg) {
+        System.out.println(DukeString.SEPARATOR);
```

Comment:
Perhaps joining the separators as a string before and after the msg and then printing the output would be better instead of calling System.out.println() multiple times?